src/styles/components/Input.scss

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2021 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+@import "./../common/mixins";
+@import "./../common/colors";
+@import "./../common/borders";
+@import "./../common/fonts";
+
+.clr-control-container {
+    font-family: $dell-font;
+    display: inline-flex;
+    margin-bottom: $input-container-margin;
+    margin-top: $input-container-margin;
+
+    input {
```

Comment:
@singha158ecs  I think we need to override `clr-input` class and not input element itself.